app/controllers/admin/schedule_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Admin
+  class ScheduleController < Admin::BaseController
+    # By authorizing 'conference' resource, we can ensure there will be no unauthorized access to
+    # the schedule of a conference, which should not be accessed in the first place
+    load_and_authorize_resource :schedule
+    load_and_authorize_resource :conference, find_by: :short_title
+    load_and_authorize_resource :program, through: :conference, singleton: true
+    load_resource :venue, through: :conference, singleton: true
+
+    skip_before_action :verify_authenticity_token, only: [:update]
+
+    def index
+      @schedules = @conference.program.schedules
```

Comment:
If  `` load_and_authorize_resource :schedule `` would work, this shouldn't be necessary!
Maybe you have another look at https://github.com/CanCanCommunity/cancancan ?